tms/models/product_template.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2024 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    is_vehicle = fields.Boolean(string="Is a Vehicle")
```

Comment:
why?

Reply:
@marcelsavegnago To create a vehicle when we buy one.

Reply:
The fleet_vehicle_stock facilitates this to some extent. If necessary, I will ask my team to migrate it to version 17.0.

Reply:
Additionally, if it's necessary to migrate other fleet modules, just let me know and I will do my best to prioritize it.

Reply:
Another comment: if you are going to implement any control of insurance or vehicle documentation, I would not implement it in TMS, but rather in a generic module within Fleet.